.scripts/dependencies.ts

```diff
@@ -222,15 +222,15 @@ function updateGeneratedPackageDependencyVersion(codeFilePath: string, dependenc
       codeFilePath,
       codeFileContents,
       dependencyName,
-      new RegExp(`\\\\"${dependencyName}\\\\": \\\\"(.*)\\\\"`),
+      new RegExp(`\\\\"${dependencyName}\\\\": \\\\"(.*?)\\\\"`),
```

Comment:
Does it mean the version can be empty?

Reply:
I suppose it could. That's okay with me though.